addon/actions.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash'
+import {validateValue} from './validator/index'
+import {aggregateResults} from './validator/utils'
+import {getPath} from './dereference'
+
+export const CHANGE_VALUE = 'CHANGE_VALUE'
+export const VALIDATION_RESOLVED = 'VALIDATION_RESOLVED'
+
+export function changeValue (bunsenId, value) {
+  return {
+    type: CHANGE_VALUE,
+    bunsenId,
+    value
+  }
+}
+
+export function updateValidationResults (validationResult) {
```

Comment:
Yeah i still need to go on a hunt and add JSDoc everywhere it is missing. Was just trying to get this 🐘 PR out to unblock @cstolli.